apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -234,7 +235,7 @@ export const ZeroPage: FC = () => {
                 <DialogBody>
                   <CloseCreditLine
                     onSubmit={handleTroveClose}
-                    creditValue={String(debt)}
+                    creditValue={String(debt - LIQUIDATION_RESERVE_AMOUNT)}
                     collateralValue={String(collateral)}
                     availableBalance={zusdBalance}
```

Comment:
This is wrong because even if I have enough `DLLR` and choose it as a token to close an LoC, I am not able to do it if I don't have enough `ZUSD`. You need to send both balances and choose which one you need to use.